src/effects/VST/VSTEffect.h

```diff
@@ -199,6 +204,33 @@ struct VSTEffectWrapper : public VSTEffectLink, public XMLTagHandler
    void SaveFXP(const wxFileName& fn) const;
    void SaveFXProgram(wxMemoryBuffer& buf, int index) const;
 
+   // VST plugin -> host callback
+   static intptr_t AudioMaster(AEffect *effect,
+                               int32_t opcode,
+                               int32_t index,
+                               intptr_t value,
+                               void * ptr,
+                               float opt);
+
+   // These are needed to move the AudioMaster callback from VSTEffect to this struct
+   //
+   static intptr_t mCurrentEffectID;
```

Comment:
I see why this moved member ever needed to be static.  Make it non-static and initialize it with `{}`

Reply:
ok

Reply:
Can’t you also rewrite the case in AudioMaster to use `vst->mCurrentEffectID`?

Reply:
That's right, then it is possible - done.